backends/dnf/notify_packagekit.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Gordon Messmer
+#
+# Licensed under the GNU Lesser General Public License Version 2.1
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+"""
+Notify packagekitd when packages are updated.
```

Comment:
This will also work when packages are installed and removed, right?

Reply:
Yes.  According to https://dnf.readthedocs.io/en/latest/api_plugins.html#dnf.Plugin.transaction this plugin would run after any transaction, and it would inform PackageKit that "StateHasChanged".

Reply:
Great. Please disambiguate the comment. ;)

Reply:
I've updated the comment and squashed commits.